recipe/meta.yaml

```diff
@@ -55,6 +55,8 @@ outputs:
         - ninja
         - protobuf {{ libprotobuf }}
         - make      # [linux]
+        - mpi4py
+        - openmpi
```

Comment:
These should not be needed. The host requires are what you wanted. That said, if the build fails to find MPI without openmpi in the build requirements it may need to be here because of bugs in the MPI cmake code.

Reply:
Fixed. Will build and test again.

Reply:
Okay, still works. Also verified with python3.8. Script was made by Pengfei and is a better indicator than the small test used before. Script runs successfully with the current commit.